From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 15 Apr 2016 16:43:02 +0200
Subject: [PATCH] basic: user-utils.c needs missing.h for secure_getenv

Otherwise building may fail with:

src/basic/user-util.c: In function 'get_home_dir':
src/basic/user-util.c:343:9: error: implicit declaration of function 'secure_getenv' [-Werror=implicit-function-declaration]
---
 src/basic/user-util.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/user-util.c b/src/basic/user-util.c
index 19155bce5364..f65ca3edaa28 100644
--- a/src/basic/user-util.c
+++ b/src/basic/user-util.c
@@ -30,6 +30,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
+#include "missing.h"
 #include "alloc-util.h"
 #include "fd-util.h"
 #include "formats-util.h"
